core/plugin_api.php

```diff
@@ -185,21 +185,17 @@ function plugin_route_group( $p_base_name = null ) {
 }
 
 /**
- * Return a path to a plugin file.
- * @param string $p_filename  File name.
- * @param string $p_base_name Plugin base name.
- * @return mixed File path or false if FNF
+ * Return a path to a plugin file(s).
```

Comment:
I find the `(s)` somewhat misleading, as the function only returns the path for a single file.

I suppose you added that to reflect the new ability to return the directory; if that's the case, then I believe it would be better to rephrase, like `Returns the path to a plugin file, or the plugin's files directory.`